gdnative-core/src/nativescript/class.rs

```diff
@@ -64,13 +64,13 @@ pub trait NativeClass: Sized + 'static {
 
     /// Register any exported properties to Godot.
     #[inline]
-    fn register_properties(_builder: &crate::init::ClassBuilder<Self>) {}
+    fn register_properties(_builder: &crate::nativescript::init::ClassBuilder<Self>) {}
```

Comment:
Just a nitpick, but maybe we can `use crate::nativescript::init::ClassBuilder` here?